env.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module 'react-native-dotenv' {
+    export const REACT_APP_GOOGLE_DIRECTIONS_API_KEY: string;
+    export const REACT_APP_GOOGLE_PLACES_API: string;
+    export const REACT_APP_GOOGLE_AUTH_ANDROID_CLIENT_ID: string;
+}
```

Comment:
From the commit message:
> Fix some typescript issues with .env file (react-native-dotenv). Need to export individual env key in an env declaration file hence the env.d.ts

It's because I get the following error when using the .env variables with dotenv
> Module '"../node_modules/@types/react-native-dotenv"' has no exported member 'REACT_APP_GOOGLE_AUTH_ANDROID_CLIENT_ID'.